glide.lock

```diff
@@ -1,22 +1,28 @@
-hash: 5fc263bfacc703e0a599166ccae968891dd53d7b4e42a3d8356b3194e45833bb
-updated: 2017-12-18T16:46:06.7851579+05:30
+hash: c48583fd7daa415c986e22638c3036b02f22698a0f5d5f583ce1b6c06833a3b3
+updated: 2018-02-28T14:36:47.873451024Z
 imports:
+- name: bitbucket.org/ww/goautoneg
+  version: 75cd24fc2f2c2a2088577d12123ddee5f54e0675
 - name: github.com/asaskevich/govalidator
-  version: 852d82c746b23d9b357b210ea470d99f4e023b72
+  version: 593d64559f7600f29581a3ee42177f5dbded27a9
```

Comment:
This hunk is probably an automatic update by glide, unrelated to the prometheus patch and should be removed from this patch.